tuf/store/filestore.go

```diff
@@ -53,7 +55,14 @@ func (f *FilesystemStore) GetMeta(name string, size int64) ([]byte, error) {
 		}
 		return nil, err
 	}
-	return meta, nil
+	if size == -1 {
```

Comment:
I have no strong opinions.  0 it's basically a no-op, and seems silly to call a function with, but still seems to be a valid value ("please do this 0 times"), where as you can't do things `-1` times.

I'm also of the opinion that consistency is more important than the exact value, though :)